.github/workflows/stale.yml

```diff
@@ -0,0 +1,32 @@
+name: Check stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
```

Comment:
I looked this up, on from my understanding, it will run the workflow, but will only log potential changes without actually making them, right?

Reply:
Yes, it is correct. It will only print the output in the workflow logs in the actions tab.

Reply:
Then fix the `name` field, and let's merge this in. We'll get far more info regarding the correctness of the job from the logs than we will from leaving the PR up for weeks and going over 30-40 lines of yml :)

Reply:
I also added a message and a label for when the PR is closed by the workflow :)